docs/UserGuide.md

```diff
@@ -206,117 +174,120 @@ Date:   Fri Feb 9 19:13:13 2018 +0800
+* Second row: repo data
+
+Here is an example:
+
+Repository's Location|Branch|File formats|Ignore Glob List|Ignore standalone config|Ignore Commits List
+---------------------|------|------------|----------------|------------------------|-------------------
+`https://github.com/foo/bar.git`|`master`|`java;css`|`test/**`|`yes`|`2fb6b9b2dd9fa40bf0f9815da2cb0ae8731436c7;c5a6dc774e22099cd9ddeb0faff1e75f9cf4f151`
+
+<br>
+
+Column Name | Explanation
+----------- | -----------
+Repository's Location | The `GitHub URL` or `Disk Path` to the git repository e.g., `https://github.com/foo/bar.git` or `C:\Users\user\Desktop\GitHub\foo\bar`
+Branch | The branch to analyze in the target repository e.g., `master`. Default: the default branch of the repo
+[Optional] File formats<sup>*</sup> | The file extensions to analyze. Default: `adoc;cs;css;fxml;gradle;html;java;js;json;jsp;md;py;tag;xml`
+[Optional] Ignore Glob List<sup>*</sup> | The list of file path globs to ignore during analysis for each author. e.g., `test/**;temp/**`
+[Optional] Ignore standalone config | To ignore the standalone config file (if any) in target repository, enter **`yes`**. If the cell is empty, the standalone config file in the repo will take precedence over configurations provided in the csv files.
+[Optional] Ignore Commit List<sup>*</sup> | The list of commits to ignore during analysis. For accurate results, the commits should be provided with their full hash.
+
+<sup>* **Multi-value column**: multiple values can be entered in this column using a semicolon `;` as the separator.</sup>
+
+#### `author-config.csv`
+
+Optionally, you can use a `author-config.csv` (which should be in the same directory as `repo-config.csv` file) to provide more details about the authors to analyze ([example](author-config.csv)). It should contain the following columns:
+
+Column Name | Explanation
+----------- | -----------
+Repository's Location | Same as `repo-config.csv`
+Branch | The branch to analyze for this author
+Author's GitHub ID | GitHub username of the target author e.g., `JohnDoe`
+[Optional] Author's Display Name | The name to display for the author. Default: author's GitHub username.
+[Optional] Author's Git Author Name<sup>*</sup> | Detailed explanation below
+[Optional] Ignore Glob List<sup>*</sup> | Files to ignore for this author, in addition to files ignored by the patterns specified in `repo-config.csv`
+
+<sup>* **Multi-value column**: multiple values can be entered in this column using a semicolon `;` as the separator.</sup>
+
+If `author-config.csv` is not given and the repo has not provide author details in a standalone config file, all the authors of the repositories will be analyzed.
+
+<hr>
+
+## Analyzing Multiple Repos
+
+This section assumes that you have read the earlier sections of the user guide.
+
+The simplest way to analyze multiple repos in one go is to use the `-repos` parameter in place of the `-repo` parameter when running RepoSense.
```

Comment:
The `-repos` and `repo` parameter is the same, no need for the extra `s` to indicate multiple repos

Reply:
That's true, but maybe we keep that as unpublished knowledge? The user will find it easier to understand if `repo` is used for one repo and `repos` is used for multiples.

Reply:
>That's true, but maybe we keep that as unpublished knowledge?

Sounds good as well 👍